src/main/java/org/team1540/robot2024/RobotContainer.java

```diff
@@ -147,18 +181,28 @@ public RobotContainer() {
         }
 
 
-        // Set up auto routines
-        autoChooser = new LoggedDashboardChooser<>("Auto Choices", AutoBuilder.buildAutoChooser());
 
         // Set up FF characterization routines
-        autoChooser.addOption(
-                "Drive FF Characterization",
-                new FeedForwardCharacterization(
-                        drivetrain, drivetrain::runCharacterizationVolts, drivetrain::getCharacterizationVelocity));
-        autoChooser.addOption(
-                "Flywheels FF Characterization",
-                new FeedForwardCharacterization(
-                        shooter, volts -> shooter.setFlywheelVolts(volts, volts), () -> shooter.getLeftFlywheelSpeed() / 60));
+        AutoManager.getInstance().addAuto(
+                new AutoCommand(
+                        "Drive FF Characterization",
+                        new FeedForwardCharacterization(
+                                drivetrain, drivetrain::runCharacterizationVolts, drivetrain::getCharacterizationVelocity
+                        )
+                )
+        );
+
+        AutoManager.getInstance().addAuto(
+                new AutoCommand(
```

Comment:
Doesn't really matter but I think making this a factory method would make it a little clearer what it does (`AutoCommand.fromExisting(String name, Command command)`)